lib/solana.cpp

```diff
@@ -67,6 +67,19 @@ json Connection::sendTransactionRequest(
   return jsonRequest("sendTransaction", params);
 }
 
+json Connection::setAirdrop(
```

Comment:
should be called requestAirdropRequest so it's not in line with the remaining code